mtest/CMakeLists.txt

```diff
@@ -219,8 +219,8 @@ subdirs (
         libmscore/tuplet
 #        libmscore/text        work in progress...
         libmscore/utils
-        mscore/workspaces
-        mscore/palette
+#        mscore/workspaces    Not worked on CI, because "Could not initialize GLX"
+#        mscore/palette       Not worked on CI, because "Could not initialize GLX"
```

Comment:
If you can't beat 'me, kill 'em ;-)

Reply:
But see also https://musescore.org/en/node/311504

Reply:
> If you can't beat 'em, kill 'em ;-)
> 
> But see also https://musescore.org/en/node/311504, so this seems not the only such case?

:)  

I think the problem is because there is no graphics card or graphics driver on the CI.

Following the link, too little information